### DISCUSSION TITLE
Welcome to apkmirror-public Discussions!

### DISCUSSION THREAD

**Discussion by archon810** [archon810/apkmirror-public#168] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by COUPON4Vitamins** [comment]
Hi can someone explain me pls why some system apps are labeled with ”(READ NOTES)"?

What does (READ NOTES) mean? I've googled everywhere and never could find anything about it. 

This app is an example:

[](https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-11-7532981-release/package-installer-11-7532981-android-apk-download/)

**Comment by COUPON4Vitamins** [comment]
https://www.apkmirror.com/apk/google-inc/package-installer/package-installer-11-7532981-release/package-installer-11-7532981-android-apk-download/

**Reply by archon810** [reply]
Hit the Notes tab, and you'll see the note about a given upload. I'm seeing that it came from the Galaxy Watch 4 for the link you sent.
